pkg/restapi/vc/operation/operations.go

```diff
@@ -92,6 +97,31 @@ func (c *Operation) createCredentialHandler(rw http.ResponseWriter, req *http.Re
 	c.writeResponse(rw, validCredential)
 }
 
+func (c *Operation) verifyCredentialHandler(rw http.ResponseWriter, req *http.Request) {
+	body, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		c.writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf("failed to read request body: %s", err.Error()))
+
+		return
+	}
+
+	verified := true
+	message := successMsg
+
+	_, _, err = verifiable.NewCredential(body)
```

Comment:
i assume we need a TODO to validate particular contexts of credentials.

Reply:
Next step is to integrate signing and validation. I will add what is required in the context at that point.